firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -47,11 +49,27 @@ class Index:
 
 # Signal to update the entry in index
 @receiver(post_save, sender=DictionaryEntry)
-def update_dictionary_entry_index(sender, instance, **kwargs):
+def request_update_dictionary_entry_index(sender, instance, **kwargs):
+    if DictionaryEntry.objects.filter(id=instance.id).exists():
+        update_dictionary_entry_index.apply_async(
+            (instance.id,),
+            link_error=link_error_handler.s(),
+            retry=True,
```

Comment:
Retries are a WIP to replace the arbitrary countdown, currently testing

Reply:
I'm sure you've explained this before but why is the timeout/retry necessary here? What is potentially not ready for execution yet?

Reply:
There is a bit of a delay between when the object is created and when it can be indexed. Everything passed to a celery task must be json serializable so only the ID of the object can be passed into the task, without the delay the Model.objects.get(id) would fail